stdlib/2/__builtin__.pyi

```diff
@@ -26,17 +26,36 @@ _T = TypeVar('_T')
 _T_co = TypeVar('_T_co', covariant=True)
 _KT = TypeVar('_KT')
 _VT = TypeVar('_VT')
+_VT_co = TypeVar('_VT_co', covariant=True)
 _S = TypeVar('_S')
 _T1 = TypeVar('_T1')
 _T2 = TypeVar('_T2')
 _T3 = TypeVar('_T3')
 _T4 = TypeVar('_T4')
 _T5 = TypeVar('_T5')
 _TT = TypeVar('_TT', bound='type')
+_TC = TypeVar('_TC', bound=Type[object])
 
 class _SupportsIndex(Protocol):
     def __index__(self) -> int: ...
 
+def runtime_checkable(cls: _TC) -> _TC: ...
```

Comment:
Why are you adding this (and Collections) to builtins? They don't exist in builtins at runtime.

Reply:
Thanks for correcting, I wasn't sure about this, the original stubs for Mapping had those.

Reply:
Well if nothing else they should be imported from typing.